app-text/multimarkdown/patches/multimarkdown-6.7.0.patchset

```diff
@@ -0,0 +1,50 @@
+From a2aaa543e764c2e92dfa30353e60debfbec857ba Mon Sep 17 00:00:00 2001
+From: Begasus <begasus@gmail.com>
+Date: Fri, 6 Oct 2023 18:35:39 +0200
+Subject: Fix installation paths
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index f51d94c..512b908 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -570,14 +570,14 @@ else (hasParent)
+ 			"${PROJECT_BINARY_DIR}/LICENSE.txt"
+ 			"${PROJECT_BINARY_DIR}/README.txt"
+ 			COMPONENT Docs
+-			DESTINATION share/doc/MultiMarkdown
++			DESTINATION ${CMAKE_INSTALL_DOCDIR}/MultiMarkdown
+ 		)
+ 		set (CPACK_COMPONENT_DOCS_DISPLAY_NAME "Documentation")
+ 		set (CPACK_COMPONENT_DOCS_DESCRIPTION "Install README and LICENSE.")
+ 
+ 		# Install LaTeX support files
+ 		install (FILES ${latex}
+-			DESTINATION share/texmf/tex/latex/mmd6
++			DESTINATION ${CMAKE_INSTALL_DATADIR}/texmf/tex/latex/mmd6
```

Comment:
Our texlive packages use `$dataDir/texlive/texmf/...` as a base directory. Maybe this needs an actual additional `/texlive/` in there to actually work.

(I didn't test it though.)

Reply:
Didn't check that part too, I just adjusted the installation path rather then moving the installed (share) to data in the recipe.
But yeah sounds about right. :)

Reply:
Working on it, set the path now to `$dataDir/texlive/texmf-dist/...` (which should reflect the layout for the other .sty files ending up in latex subdirectory).